devtools/gradle-it/build.gradle

```diff
@@ -0,0 +1,23 @@
+apply plugin: 'java'
+apply plugin: 'io.quarkus.gradle.plugin'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath fileTree(dir: '../gradle/build/libs', include: 'quarkus-gradle-plugin-999-SNAPSHOT.jar')
+        classpath fileTree(dir: '../gradle/target/dependencies/compile', include: '*.jar')
+    }
+}
+
+repositories {
+     mavenCentral()
+     maven { url 'http://repository.jboss.org/nexus/content/groups/public' }
```

Comment:
not sure if this line is needed?

Reply:
You're right it's not, so I've removed it.